src/controllers/recipe.controller.ts

```diff
@@ -97,11 +101,9 @@ export class RecipeController {
       }
 
       if (updatedRecipe.OK && updatedRecipe.recipe) {
-        webhookService.webhookHandler(
-          userId,
-          "update_recipe",
-          updatedRecipe.recipe
-        );
+        const jsonedRecipe = updatedRecipe.recipe?.toJSON();
+        jsonedRecipe._id = jsonedRecipe._id.toString();
+        webhookService.webhookHandler(userId, "update_recipe", jsonedRecipe);
```

Comment:
Please rename `webhookHandler` function, it doesn't say what it does.
The handler would be used for "handling" something, this seems more like "sendWebhookNotification"